src/components/CrazyInput/CrazyInput.vue

```diff
@@ -167,16 +167,26 @@
         >
           <!-- Chip selection -->
           <PawChip
-            v-for="item in chips"
+            v-for="item in maxChips>0 ? chips.slice(0 ,maxChips) : chips"
```

Comment:
this condition will always be true as maxChips has default value of 2, remove condition

Reply:
I would disagree, this condition allows us to show all chips if needed (default value updated to 0 - good catch)